packages/web-sdk/src/index.ts

```diff
@@ -28,11 +28,6 @@ export default class Integraflow {
         if (!this.client) {
             this.client = new Integraflow(config);
             this.initialized = true;
-        } else {
```

Comment:
@AfeezGL Why did you remove this?

Reply:
@jdamilola This else block was added to `Integraflow` class to support context updates in the preview panel, this block has now been moved to the internal `_Integraflow` class.